packages/rich-text-links/src/index.ts

```diff
@@ -82,7 +83,7 @@ function isContentNode(node: Node): node is Inline | Block {
   return 'content' in node && Array.isArray(node.content);
 }
 
-function visitNodes(startNode: Node, onVisit: (node: Node) => void): void {
+function visitNodes(startNode: Maybe<Node>, onVisit: (node: Node) => void): void {
```

Comment:
Any reason why we don't need the maybe on `onVisit`?

Reply:
Any nullable values are handled on line 90 and will not be passed along to `onVisit`. This prevents consumers of this method to add additional null checks their handler functions